autoscheduler/frontend/src/redux/actions/courseCards.ts

```diff
@@ -311,7 +315,7 @@ export function replaceCourseCards(
     courseCards.forEach((courseCard, idx) => {
       const deserializedCard = deserializeCourseCard(courseCard);
       deserializedCards.push(deserializedCard);
-      dispatch(updateCourseCardSync(idx, deserializedCard));
+      dispatch(addCourseCard(deserializedCard, idx));
```

Comment:
What's the thought process of the change from `updateCourseCardSync` to `addCourseCard`? Is there a reason it was `updateCourseCardSync` in the first place?

Reply:
Since I made `updateCourseCard` not change state unless the card already exists (which is how it's no longer possible for deleted cards to pop into existence), the call that originally creates the cards can't use it anymore